operations/finance.md

```diff
@@ -519,6 +521,120 @@ Returns all bills \(both receipts and invoices\) that have been closed in the sp
+        }
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| CreditCards | array of [Credit card](finance.md#credit-card)s | required | The credit cards. |
+
+
+## Get all credit cards by customers
+
+Returns all credit cards of specified customers.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/creditCards/getAllByCustomers`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CustomerIds": [
+        "a3c90426-43f2-4b53-8482-446dfc724bd2"
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CustomerIds` | array of string | required | Unique identifier of the [Customer](customers.md#customer). |
+
+### Response
+
+```javascript
+{
+    "CreditCards": [
+        {
+            "CreatedUtc": "2018-05-24T13:45:29Z",
+            "CustomerId": "a3c90426-43f2-4b53-8482-446dfc724bd2",
+            "Expiration": "2020-11",
+            "Format": "Physical",
+            "Id": "f1d94a32-b4be-479b-9e47-a9fcb03d5196",
+            "IsActive": true,
+            "Kind": "Gateway",
+            "ObfuscatedNumber": "************1111",
+            "State": "Enabled",
+            "Type": "Visa"
+        }
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| CreditCards | array of [Credit card](finance.md#credit-card)s | required | Credit cards of the specified [Customer](customers.md#customer)s. |
+
+#### Credit card
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the credit card. |
+| `CustomerId` | string | required | Unique identifier of the credit card owner. |
+| `CreatedUtc` | string | required | Creation date and time of the credit card. |
+| `Expiration` | string | optional | Expiration of the credit card. |
+| `IsActive` | boolean | required | Whether the credit card is still active. |
+| `ObfuscatedNumber` | string | optinal | Obfuscated credit card number. At most first six digits and last four digits can be specified, the digits in between should be replaced with `*`. |
+| `Format` | string | required | Format of the credit card, one of the: `Physical`, `Virtual`. |
+| `Kind` | string | required | Kind of the credit card, one of the: `Terminal`, `Gateway`. |
+| `State` | string | required | State of the credit card, one of the: `Enabled`, `Disabled`. |
+| `Type` | string | required | Type of the credit card, one of the: `MasterCard`, `Visa`, `Amex`, `Discover`, `DinersClub`, `Jcb`, `Maestro`, `UnionPay`, `VPay`, `RuPay`, `Dankort`, `Mir`, `Verve`, `Troy`, `PostFinance`, `Giro`, `Bancomat`, `Bc`, `CarteBleue`, `Eftpos`, `Eps`, `Interac`, `Isracard`, `Meps`, `Nets`. |
```

Comment:
Types for enums. for `Type` enum, mention only Major ones and last item should be `...`